### DISCUSSION TITLE
error: the user '_nixbld1' in the group 'nixbld' does not exist

### DISCUSSION THREAD

**Discussion by JackPala** [nix-community/raspberry-pi-nix#84] [Q&A]
I get this error when I try to build the example

```
error: the user '_nixbld1' in the group 'nixbld' does not exist
```

I clone the repo and set the wifi controller in the flake and then run the specified command and that is the output to the terminal.

Building from Nix darwin on MacOS ARM

**Comment by tstat** [comment]
This isn't an error related to this repo -- it looks like you have a broken nix install. I recommend seeking help on the nix discourse or matrix.
